packages/kotti-ui/source/kotti-field-text-area/KtFieldTextArea.vue

```diff
@@ -27,9 +31,26 @@ export default defineComponent({
 			props,
 			supports: KOTTI_FIELD_TEXT_AREA_SUPPORTS,
 		})
+		const textareaRef = ref<HTMLTextAreaElement | null>(null)
 
 		const { forceUpdate, forceUpdateKey } = useForceUpdate()
 
+		const resizeTextarea = (
+			textarea: HTMLTextAreaElement | null,
+			autoSize: boolean,
+			maxHeightInPixels: number | null,
+		) => {
+			if (
+				autoSize &&
+				textarea &&
+				((maxHeightInPixels && textarea?.scrollHeight < maxHeightInPixels) ||
+					!maxHeightInPixels)
```

Comment:
You should first check if `maxHeightInPixels` is a valid number. 
This can be rewritten just as:
```suggestion
				(!maxHeightInPixels || textarea.scrollHeight < maxHeightInPixels)
```
The OR `||` operator will break on the first `truthy` evaluation.

Also, you don't need the optional chaining operator in:
```
textarea?.scrollHeight
```
because you are already checking `textarea` in line 45